runtime/lua/vscode/internal.lua

```diff
@@ -280,4 +282,71 @@ function M.wslpath(path)
   return vim.trim(ret)
 end
 
+--#region Buffer management
+
+local function set_buffer_autocmd(buf)
```

Comment:
```suggestion
--- Implements :write and related commands, via buftype=acwrite. #521 #1260
local function set_buffer_autocmd(buf)
```

Reply:
Incorrect, and this comment has already been added.